src/utils/vscode/summarizeCodeContext.ts

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
```

Comment:
Is this function used anywhere?

Reply:
Currently in the process of re-introducing it to the new UI